llama_index/agent/openai_agent.py

```diff
@@ -53,6 +55,38 @@ def call_function(
     )
 
 
+class StreamingChatResponse:
+    """Streaming chat response to user and writing to chat history."""
+
+    def __init__(self, chat_stream: ChatResponseGen) -> None:
+        self.chat_stream = chat_stream
+        self.queue: Queue = Queue()
+        self.is_done = False
+        self.response = ""
```

Comment:
what is this for?

Reply:
Some compat to make it similar to our other response objects (whoops, forgot to add `__str__`